collector/monitors/slashing_monitor.go

```diff
@@ -0,0 +1,125 @@
+package monitors
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/lidofinance/terra-monitors/openapi/client/transactions"
+
+	"github.com/lidofinance/terra-monitors/collector/config"
+	"github.com/lidofinance/terra-monitors/openapi/client"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	SlashingNumJailedValidators     Metric = "slashing_num_jailed_validators"
+	SlashingNumTombstonedValidators Metric = "slashing_num_tombstoned_validators"
+	SlashingNumMissedBlocks         Metric = "slashing_num_missed_blocks" // TODO: add a "validator_address" label.
+)
+
+const (
+	jailTimeLayout = "2006-01-02T15:04:05Z"
+)
+
+type SlashingMonitor struct {
+	metrics   map[Metric]float64
+	apiClient *client.TerraLiteForTerra
+	logger    *logrus.Logger
+}
+
+func NewSlashingMonitor(cfg config.CollectorConfig) *SlashingMonitor {
+	m := &SlashingMonitor{
+		metrics:   make(map[Metric]float64),
+		apiClient: cfg.GetTerraClient(),
+		logger:    cfg.Logger,
+	}
+
+	return m
+}
+
+func (m *SlashingMonitor) Name() string {
+	return "Slashing"
+}
+
+func (m *SlashingMonitor) InitMetrics() {
+	m.metrics = map[Metric]float64{
+		SlashingNumJailedValidators:     0,
+		SlashingNumTombstonedValidators: 0,
+		SlashingNumMissedBlocks:         0,
+	}
+}
+
+func (m *SlashingMonitor) Handler(ctx context.Context) error {
+	m.InitMetrics()
+
+	validatorAddresses, err := m.getValidatorPublicKeys()
```

Comment:
Variable is called validator**Addresses** but a function is supposed to return public keys. Also similar typos below because of this.